---
layout: paper

name: "Leveraging a pharmacogenomics knowledgebase to formulate a drug response phenotype terminology for genomic medicine"
date: "2022-12-01"
authors: "Yiqing Zhao, Matthew Brush, Chen Wang, Alex H Wagner, Hongfang Liu, Robert R Freimuth"
journal_cite: "Bioinformatics, Volume 38, Issue 23, 1 December 2022, Pages 5279–5287"
journal: Bioinformatics
volume: 38
issue: 23
pages: "5279-5287"
doi: "https://doi.org/10.1093/bioinformatics/btac646"
pmid: "36222570"
publisher_url: "https://academic.oup.com/bioinformatics/article/38/23/5279/6759370"

thumbnail: "pgx-kb.jpeg"

projects:
software:
---
### Motivation

Despite the increasing evidence of utility of genomic medicine in clinical practice, systematically integrating genomic medicine information and knowledge into clinical systems with a high-level of consistency, scalability and computability remains challenging. A comprehensive terminology is required for relevant concepts and the associated knowledge model for representing relationships. In this study, we leveraged PharmGKB, a comprehensive pharmacogenomics (PGx) knowledgebase, to formulate a terminology for drug response phenotypes that can represent relationships between genetic variants and treatments. We evaluated coverage of the terminology through manual review of a randomly selected subset of 200 sentences extracted from genetic reports that contained concepts for ‘Genes and Gene Products’ and ‘Treatments’.

### Results

Results showed that our proposed drug response phenotype terminology could cover 96% of the drug response phenotypes in genetic reports. Among 18 653 sentences that contained both ‘Genes and Gene Products’ and ‘Treatments’, 3011 sentences were able to be mapped to a drug response phenotype in our proposed terminology, among which the most discussed drug response phenotypes were response (994), sensitivity (829) and survival (332). In addition, we were able to re-analyze genetic report context incorporating the proposed terminology and enrich our previously proposed PGx knowledge model to reveal relationships between genetic variants and treatments. In conclusion, we proposed a drug response phenotype terminology that enhanced structured knowledge representation of genomic medicine.

